src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -220,7 +232,7 @@ private void runOnDocker(DumbSlave s) throws Exception {
         do {
             Thread.sleep(1000);
             baos = new ByteArrayOutputStream();
-            assertEquals(0, dockerLauncher.launch().cmds("ps", "-e", "-o", "pid,stat,command").stdout(new TeeOutputStream(baos, System.out)).join());
+            assertEquals(0, dockerLauncher.launch().cmds("ps", "-e", "-o", "pid,stat,comm").stdout(new TeeOutputStream(baos, System.out)).join());
```

Comment:
`comm` appears to be the standardized name: http://pubs.opengroup.org/onlinepubs/009695399/utilities/ps.html, and BusyBox doesn't support `command`.